ci/ray_ci/test_utils.py

```diff
@@ -84,11 +102,24 @@ def test_filter_out_flaky_tests(mock_gen_from_s3):
             _make_test("darwin://test_4", "passing", "ci"),
         ],
     )
-
     test_targets = ["//test_1", "//test_2", "//test_3", "//test_4"]
     output = io.StringIO()
-    filter_out_flaky_tests(io.StringIO("\n".join(test_targets)), output, "darwin:")
-    assert output.getvalue() == "//test_3\n//test_4\n"
+
+    # Test invalid filter option
+    if filter_option not in ["only-flaky", "non-flaky"]:
```

Comment:
could you write another unit test function for this? so that this unit test is only for valid input? and we do not need to condition on the parameter here?

Reply:
done!